golem-worker-bridge/src/oas_worker_bridge.rs

```diff
@@ -74,7 +87,7 @@ fn get_route_from_path_item(
 
     let worker_bridge_info = path_item
         .extensions
-        .get("x-worker-bridge")
+        .get("x-golem-worker-bridge")
```

Comment:
it will be good to have constant

Reply:
will not be more appropriate to 
x-golem-worker-bridge ->  x-golem-worker-binding
?